validator/server_arb/validator_spawner.go

```diff
@@ -105,7 +105,7 @@ func (s *ArbitratorSpawner) WasmModuleRoots() ([]common.Hash, error) {
 }
 
 func (s *ArbitratorSpawner) StylusArchs() []rawdb.WasmTarget {
-	return []rawdb.WasmTarget{rawdb.TargetWavm}
+	return []rawdb.WasmTarget{rawdb.TargetWavm, rawdb.LocalTarget()}
```

Comment:
Not sure if this should be this actually 🤔 
```suggestion
	return []rawdb.WasmTarget{rawdb.LocalTarget()}
```